content/en/marketplace/marketplace.md

```diff
@@ -0,0 +1,192 @@
+`ddev config set marketplace /path/to/marketplace`
+
+`ddev config set repo marketplace`
+
+#### Integrations-Extras:
+`ddev config set extras /path/to/integrations-extras`
+
+`ddev config set repo extras`
+
+#### Tile Only Listing
+For standalone software and services--or if your integration is using the Datadog API and does not contain any Python code--the Development Toolkit supports a tile-only command. 
+`ddev create -t` 
+
+#### Full Integration
+For the complete integration scaffolding run: 
+
+`ddev create`
+
+### 5. Populate the Tile Scaffolding
+
+The ddev commands in the previous section generate a skeleton of folders and files that make up your tile assets:
+
+#### README.md
+
+* Include “Overview,” “Setup,” and “Support” sections with H2 headings (## in Markdown).
+* The “Overview" heading should clearly describe the value your offering provides users, as well as how it can be used together with Datadog for more comprehensive observability. We recommend adding images of your software or dashboards in action. This section will be displayed in the “Overview” tab of your tile.
+* The “Setup” heading should provide straightforward configuration steps for users to install or use your offering. This section will be displayed in the “Configuration” tab of your tile.
+* The “Support” heading should identify a contact for support, and possibly an option to submit product feedback. This section will be displayed in the “Support” tab of your tile. 
+
+#### Images
+* Store all images used in your README.md file in this folder. 
+
+#### Manifest.json
+* Contains multiple JSON objects, such as display_name, public_title, and author. 
+* More information about manifest.json fields can be found in our [Integrations Assets Reference](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file)
+* Details on the pricing object are described in the private [Marketplace README](https://github.com/DataDog/marketplace/blob/master/README.md#faq).
+
+#### Metadata.csv
+* Contains a list of the out-of-the-box metrics included in an integration, including the metric name, type, interval, and unit. 
+* More information about metadata.csv fields can be found in our [Integrations Assets Reference](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file). 
+* **Note:** All Marketplace metrics count as custom metrics. 
+
+#### Dashboards and Monitors
+* Contain the out-of-the-box dashboards and monitors (alerts) for your integration. 
+* You can create dashboards and monitors directly in your sandbox account and export them as JSON files. 
+* See our [Dashboarding Best Practices](https://datadoghq.dev/integrations-core/guidelines/dashboards/) document for details.
+
+#### Logo
+* Consists of an SVG, which our DesignOps team implements throughout the Datadog app for both light and dark modes. 
+* **Note:** Technology Partners are responsible for the licensing of the logos they submit.  
+
+#### Changelog.md
```

Comment:
Can we add a tiny note that this should be in descending order (latest version at the top)

Reply:
Thanks for the suggestion, Nick. This has been added.